src/components/CustomerQuota/AddUserModal.tsx

```diff
@@ -143,6 +149,11 @@ export const AddUserModal: FunctionComponent<AddUserModal> = ({
             onBarCodeScanned={onBarCodeScanned}
             onCancel={() => setShouldShowCamera(false)}
             cancelButtonText="Enter ID manually"
+            barCodeTypes={
```

Comment:
Hmm similar comment here, should default be code39 instead haha

Reply:
agreed!